lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -619,8 +624,14 @@ void TypeAnalyzer::fillInDTypeGivenDTypeAndDataType(ValueKnowledge &knowledge,
   fillInDTypeGivenDTypeIntAndInputDType(knowledge, dtype, dtypeForDataType);
 }
 
-ChangeResult TypeAnalyzer::visitOperation(
-    Operation *op, ArrayRef<LatticeElement<ValueKnowledge> *> operands) {
+void TypeAnalyzer::visitOperation(Operation *op,
+                                  ArrayRef<const ValueState *> operandsConst,
+                                  ArrayRef<ValueState *> results) {
+  SmallVector<ValueState *> nonConstOperands;
```

Comment:
Is there a need to make these non constant other than propagating the `const` to other methods? If not, I think it is cleaner to add `const` instead of doing this.